src/app/components/icons/IconBrowser.tsx

```diff
@@ -333,7 +333,8 @@ export const IconBrowser: React.FC = (): JSX.Element => {
                                                             selected={
                                                                 focusedIcon &&
                                                                 getMuiIconName(focusedIcon.name) ===
-                                                                    getMuiIconName(icon.name)
+                                                                    getMuiIconName(icon.name) &&
```

Comment:
You probably also need to add a param to the query string for which icon class (material or blue) the icon is. Otherwise, we can never deep link to the MUI apple icon.

Reply:
We can default to either one if none is specified.

Reply:
My implementation: 

`icon=Apple&isMaterial=` | `true` | `false` | `asdf` | _none_
-|-|-|-|-
PX Blue Apple |                 | selected | selected | selected
Material Apple | selected |                |                 | 

`icon=Apps&isMaterial=` | `true` | `false` | `asdf` | _none_
-|-|-|-|-
Material Apps | selected |                |                 | selected